src/index.js

```diff
@@ -152,7 +152,7 @@ function createFilteredErrorRule(filter: CollectOutputElement => any) {
   };
 }
 
-export default {
+module.exports = {
```

Comment:
@amilajack This change broke the npm package:

```sh
> require('eslint-plugin-flowtype-errors')
undefined
```

This can be fixed by either using `export default` again, or by changing the `main` field in `package.json` to point to `dist/index.js`. Not sure which one you'd prefer.

Reply:
Oh right. I'll fix that now. My bad